[2024-06-28T12:36:39.896+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T12:36:40.248+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.fetch_weather_data manual__2024-06-28T12:36:25.437793+00:00 [queued]>
[2024-06-28T12:36:40.265+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.fetch_weather_data manual__2024-06-28T12:36:25.437793+00:00 [queued]>
[2024-06-28T12:36:40.266+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-28T12:36:40.301+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_weather_data> on 2024-06-28 12:36:25.437793+00:00
[2024-06-28T12:36:40.334+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=217) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-28T12:36:40.337+0000] {standard_task_runner.py:63} INFO - Started process 223 to run task
[2024-06-28T12:36:40.345+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'fetch_weather_data', 'manual__2024-06-28T12:36:25.437793+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpje2ku5ab']
[2024-06-28T12:36:40.356+0000] {standard_task_runner.py:91} INFO - Job 85: Subtask fetch_weather_data
[2024-06-28T12:36:40.552+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_data_pipeline.fetch_weather_data manual__2024-06-28T12:36:25.437793+00:00 [running]> on host 8429acefe19a
[2024-06-28T12:36:40.839+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='fetch_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-28T12:36:25.437793+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-28T12:36:25.437793+00:00'
[2024-06-28T12:36:40.841+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T12:36:40.952+0000] {python.py:237} INFO - Done. Returned value was: new_data.csv
[2024-06-28T12:36:40.953+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T12:36:41.006+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=fetch_weather_data, run_id=manual__2024-06-28T12:36:25.437793+00:00, execution_date=20240628T123625, start_date=20240628T123640, end_date=20240628T123641
[2024-06-28T12:36:41.078+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-28T12:36:41.188+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-28T12:36:41.196+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
